tests/Feature/ApiTokenExpirationWarningTest.php

```
@@ -127,4 +127,14 @@ function createTokenExpiring(User $user, Team $team, ?Carbon $expiresAt, ?Carbon
 
         Notification::assertNothingSent();
     });
+
+    test('skips tokens whose owner is no longer a team member', function () {
+        $token = createTokenExpiring($this->user, $this->team, now()->addHours(12));
```

PHPStan (2.2.7)

[warning] Parameter `#3` $expiresAt of function createTokenExpiring expects Carbon\Carbon|null, Carbon\CarbonInterface given.


(argument.type)